commercetools-models/src/main/java/io/sphere/sdk/products/commands/updateactions/StagedProductUpdateActionImpl.java

```diff
@@ -0,0 +1,21 @@
+package io.sphere.sdk.products.commands.updateactions;
+
+import io.sphere.sdk.commands.UpdateActionImpl;
+
+import javax.annotation.Nullable;
+
+abstract class StagedProductUpdateActionImpl<T> extends UpdateActionImpl<T> {
+
+    @Nullable
+    protected final Boolean staged;
+
+    protected StagedProductUpdateActionImpl(final String action, final Boolean staged) {
```

Comment:
One last issue: please add `@Nullable` in font of the `final Boolean staged` parameter.